src/pages/OurTeamPage.tsx

```diff
@@ -1,9 +1,32 @@
+import { Avatar, Divider, Space, Typography } from 'antd';
 import React from 'react';
+import { content, MemberCardProps } from '../content/OurTeamContent';
+import styles from '../styles/OurTeamPage.module.less';
+
+const MemberCard: React.FC<MemberCardProps> = (props) => {
+  return (
+    <div className={styles.ourteampage_member}>
+      <Avatar className={styles.ourteampage_avatar} size={150} src={props.imgSrc} />
+      <div className={styles.ourteampage_text}>
+        <Typography.Title level={3}>{props.title}</Typography.Title>
+        <Typography.Text>{props.subtitle}</Typography.Text>
+      </div>
+    </div>
+  );
+};
```

Comment:
This is good. I am wondering if there is anything that could be changes with the `less` so that it works on a mobile device. Right now the text gets cut off.

Reply:
After the meeting on March 30th, we decided to focus more on the desktop version, so as of for now, I'll only take into consideration for desktop version and in later iterations of the sprint, we'll look into the mobile design. Is that fine? @IanSaucy